config/locales/en.yml

```diff
@@ -34,8 +34,8 @@ en:
     branching_title: 'Branching point %{branching_number}'
     confirmation_email_subject: "Your submission to ‘%{service_name}’ {{reference_number_placeholder}}"
     confirmation_email_body: "Thank you for your submission to ‘%{service_name}’. {{reference_number_placeholder}}\n\r\nA copy of the information you provided is attached to this email."
-    reference_number_sentence: "Your reference number is {{reference_number}}."
-    reference_number_subject: "Reference number: {{reference_number}}."
+    reference_number_sentence: "Your reference number is: {{reference_number}}."
+    reference_number_subject: "reference number: {{reference_number}}"
```

Comment:
Do we also need a comma adding somewhere for the email subject? See Marks conmment from spreadsheet:

>With reference numbers enabled, the email subject lines are shown as:
Submission from form name Reference number: {{reference_number}}. 

> This should be:
Submission from form name, reference number: {{reference_number}}

Reply:
@chrispymm thanks for that. have updated